honeycombio/honeycomb-opentelemetry-swift
# feat(swift): implement Honeycomb.configure and add smoke tests

## Which problem is this PR solving?

* Adds the `Honeycomb.configure` method, which configures OpenTelemetry for Swift using `HoneycombOptions`.
* Also adds smoke tests.

## Short description of the changes

The smoke-test is based on the one in honeycomb-opentelemetry-android, as we have many of the same constraints. The iOS simulator must be run on macOS, and cannot be run in docker. However, we should still be able to use CircleCI, with a machine image.

## How to verify that this has the expected result

Follow setup instructions in DEVELOPING.md and then run:

```sh
make smoke
make unsmoke
```

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207992777792927
